comment_help.md

```diff
@@ -8,7 +8,7 @@ description: "Guide on providing feedback"
 
 **Review the documentation.** No account is needed to review the updated version of NIST SP 800-63-3. Simply follow [this link](https://github.com/usnistgov/800-63-3) and enjoy at your leisure. However, if you wish to comment you must…
```

Comment:
Yes, that will be written in the landing page, not here.  By the time you get here, you are asking how to create a GitHub issue, which I presume means the reader is heading down the GH approach ::)